lib/danger/plugin_support/plugin.rb

```diff
@@ -17,6 +17,8 @@ def self.instance_name
 
     # Since we have a reference to the Dangerfile containing all the information
     # We need to redirect the self calls to the Dangerfile
+
+    # rubocop:disable Style/MethodMissing
     def method_missing(method_sym, *arguments, &block)
```

Comment:
in the next release, I'm not adding something I don't understand entirely an hour or two before release.